### ISSUE TITLE
Enhancement of Copy/Paste Content on Editor

### ISSUE THREAD

**Issue by ckdanny** [streetwriters/notesnook#3313]
### What problem are you facing?

Markdown formatted text can only be pasted as paint text, which lost all the formatting in the Notesnook. Also, no way to copy the formatted text in Notesnook to be markdown format. 

### What's the solution you'd like to see in Notesnook?

It would be great to enable users to copy editor content as either (1) Markdown or (2) Rich Text, allowing them to paste the content into webpages or other applications. Additionally, providing the ability to paste content from Markdown formatted text would be advantageous as well.

**Comment by thecodrr** [comment]
This is really hard to get right. How would the app know that you are pasting as Markdown? There could be a special shortcut to copy & paste as markdown.

**Comment by ckdanny** [comment]
I think that makes sense to have a special shortcut, may be `⌘⌥V` for pasting from markdown, similarly, `⌘⌥C` can copy as markdown.

**Comment by Littleweisheit** [comment]
this is annoying

**Comment by ajkdrag** [comment]
This is a very important feature. I don't understand why this is missing. A lot of people use ChatGPT now and it outputs in markdown. Without this feature, can't copy past to Notesnook. Please think about this. @thecodrr  Even a basic solution that works for simple markdown (commontext) would be fine. An alternative (for now) is the importer, but it is a hectic process: copying from chatgpt, pasting to file, uploading to the importer, opening notesnook, importing the zip, renaming/reorganizing etc. Hope you understand this.

**Comment by minecraftchicken** [comment]
Bump. This is annoying as hell

**Comment by aproposnix** [comment]
Please improve the markdown interoperability. https://monogr.ph/66fa61d339a089397a134f2c

**Comment by odeemi** [comment]
Yep. Some way to paste markdown would be a very important addition.
